CONTRIBUTING.rst

```diff
@@ -155,18 +114,91 @@ Contributing to the documentation
 
 The documentation of equistore is written in reStructuredText (rst) and uses the
 `sphinx`_ documentation generator. In order to modify the documentation, first
-create a local version of the code your machine as described above. Then, you
+create a local version of the code on your machine as described above. Then, you
 can build the documentation with:
 
 .. code-block:: bash
 
     tox -e docs
 
-You can visualise the local documentation with your favorite browser (here
+You can visualize the local documentation with your favorite browser (here
 Mozilla Firefox is used)
 
 .. code-block:: bash
 
     firefox docs/build/html/index.html
 
 .. _`sphinx` : https://www.sphinx-doc.org/en/master/
+
+Python doc strings
+~~~~~~~~~~~~~~~~~~
+
+Our docstring format follows the `sphinx format`_ and a typical function doc string
+looks like the following.
+
+.. code-block:: rst
```

Comment:
Do you think it would be useful to include the actual function here too, so the context is complete?

For instance:
```py
def func(value_1: float, value_2: int) -> float:
    """
    <insert docstring here>
    """
    ...
    return result
```

Reply:
On this note we could also highlight somewhere in this contributing documentation that contributors should use Python typing in the function arguments, indicate return types etc.